libs/gi/db/src/Database/DataManagers/TeamDataManager.ts

```diff
@@ -119,3 +119,7 @@ function validateTeam(
     lastEdit,
   }
 }
+
+export function getActiveTeamCharId(team: Team) {
```

Comment:
Can you attach this function to `TeamDataManager`? so its usage should be `database.teams.getActiveTeamCharId(teamId)`

Reply:
In the places this function is called, the `team` object is already fetched. Isn't your suggestion requiring a redundant fetch of data?

Reply:
Yes, but I think standardizing the APIs is more important than a single fetch (which is just a lookup)

Reply:
Done